Только в /root/dk_scripts/tmp/1: 6-Jan-2014072506LINK-I-Upfa2KTCS4-SW1-48_192.168.22.71.conf
Только в /root/dk_scripts/tmp/1: 8-Feb-2012182712LINK-W-Downfa8KTCS2-SW1-48_192.168.22.84.conf
diff -r /root/dk_scripts/tmp/2/LBR31-SW4-48_192.168.22.203.conf /root/dk_scripts/tmp/1/LBR31-SW4-48_192.168.22.203.conf
292c292
< name WiFi_Beeline_lexus 
---
> name Tank_FreeWiFi 
Только в /root/dk_scripts/tmp/2: TCS2-SW1-48_192.168.22.84.conf
Только в /root/dk_scripts/tmp/2: TCS4-SW1-48_192.168.22.71.conf
